src/lib/i18n/en.json

```diff
@@ -183,7 +183,8 @@
   "routes": {
     "frontpage": {
       "title": "Frontpage",
-      "footer": "{{users:number}} active {{users; 1:user; default:users;}}"
+      "footer": "{{users:number}} active {{users; 1:user; default:users;}}",
+      "endFeed": "You have reached the end of {{community_name; undefined:the feed.; default:{{community_name}}.}}"
```

Comment:
This is the cleanest way I could come up with.

I put it under routes.frontpage since that's where the old footer is too.
![image](https://github.com/user-attachments/assets/221419de-5297-41f6-aee0-60453d7172bf)